python/django/Makefile

```diff
@@ -4,4 +4,4 @@ start:
 
 deps:
 	pip3 install --upgrade setuptools
-	pip3 install django==3.1.4 django-cockroachdb==3.1.2 psycopg2
+	pip3 install django==3.1.8 django-cockroachdb==3.1.3 psycopg2
```

Comment:
looks like we can use django 3.2 quite soon https://github.com/cockroachdb/django-cockroachdb/pull/145

Reply:
All set. You could use `pip3 install django==3.2.* django-cockroachdb==3.2.* psycopg2` to always get the latest bug fix releases of each series.

Reply:
Done